tests/functional/helmholtz/kh_2d_100_x_100.py

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)